pkg/snapstore/snapstore_suite_test.go

```diff
@@ -27,8 +27,9 @@ import (
 )
 
 var (
-	testObj    *testing.T
-	swiftStore snapstore.SnapStore
+	testObj            *testing.T
+	swiftStore         snapstore.SnapStore
+	networkTimeoutFlag bool = false
```

Comment:
We will need eventually simulate other errors as well such as different HTTP errors, latency etc.

This flag is better kept in the fakePolicy or the errorPolicy I proposed rather than as a global flag.